### PR TITLE
Added the C# and Python code snippets for Relative Locators

### PR DESCRIPTION
**Pull request by nainappa**
### Description
This PR contains addition of C# and Python code snippets for Relative Locators functionality. These changes were made to all the available locales. Please review the PR and comment for any additional changes.

### Motivation and Context
Gives the better understanding of Relative Locators functionality.

### Types of changes

- [x] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

Before:
--------------------------------------------------------------------------------------------------------------
image
image

After:
--------------------------------------------------------------------------------------------------------------
image
image

### CODE REVIEW

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md [hunk 1]
```diff
@@ -327,10 +343,16 @@ WebElement submitButton= driver.findElement(withTagName("button")
                                             .toRightOf(cancelButton));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+cancelButton = driver.find_element_by_id("cancel");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md:347]:
```suggestion
cancelButton = driver.find_element(By.ID,"cancel");
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md [hunk 2]
```diff
@@ -356,10 +378,16 @@ WebElement emailAddressField = driver.findElement(withTagName("input")
                                                   .near(emailAddressLabel));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+emailAddressLabel = driver.find_element_by_id("lbl-email");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md:382]: Same as above

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md [hunk 3]
```diff
@@ -297,10 +307,16 @@ WebElement cancelButton= driver.findElement(withTagName("button")
                                             .toLeftOf(submitButton));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+submitButton = driver.find_element_by_id("submit");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md:311]:
```suggestion
submitButton = driver.find_element(By.ID,"submit");
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md [hunk 4]
```diff
@@ -267,10 +272,15 @@ WebElement passwordField = driver.findElement(withTagName("input")
 	                                          .below(emailAddressField));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+emailAddressField = driver.find_element_by_id("email");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md:276]:
```suggestion
emailAddressField = driver.find_element(By.ID, "email");
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md [hunk 5]
```diff
@@ -237,10 +237,15 @@ WebElement emailAddressField = driver.findElement(withTagName("input")
                                                   .above(passwordField));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+passwordField = driver.find_element_by_id("password");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md:241]:
```suggestion
passwordField = driver.find_element(By.ID, "password");
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.es.md [hunk 6]
```diff
@@ -359,10 +381,16 @@ WebElement emailAddressField = driver.findElement(withTagName("input")
                                                   .near(emailAddressLabel));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+emailAddressLabel = driver.find_element_by_id("lbl-email");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.es.md:385]:
```suggestion
emailAddressLabel = driver.find_element(By.ID, "lbl-email");    
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md [hunk 7]
```diff
@@ -245,10 +245,15 @@ WebElement emailAddressField = driver.findElement(withTagName("input")
                                                   .above(passwordField));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+passwordField = driver.find_element_by_id("password");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md:249]:
```suggestion
passwordField = driver.find_element(By.ID,"password");
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md [hunk 8]
```diff
@@ -275,10 +280,15 @@ WebElement passwordField = driver.findElement(withTagName("input")
 	                                          .below(emailAddressField));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+emailAddressField = driver.find_element_by_id("email");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md:284]:
```suggestion
emailAddressField = driver.find_element(By.ID, "email");
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md [hunk 9]
```diff
@@ -305,10 +315,16 @@ WebElement cancelButton= driver.findElement(withTagName("button")
                                             .toLeftOf(submitButton));                 
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+submitButton = driver.find_element_by_id("submit");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md:319]:
```suggestion
submitButton = driver.find_element(By.ID, "submit");
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md [hunk 10]
```diff
@@ -335,10 +351,16 @@ WebElement submitButton= driver.findElement(withTagName("button")
                                             .toRightOf(cancelButton));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+cancelButton = driver.find_element_by_id("cancel");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md:355]:
```suggestion
cancelButton = driver.find_element(By.ID, "cancel");
```

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md [hunk 11]
```diff
@@ -364,10 +386,16 @@ WebElement emailAddressField = driver.findElement(withTagName("input")
                                                   .near(emailAddressLabel));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+#from selenium.webdriver.support.relative_locator import with_tag_name
+emailAddressLabel = driver.find_element_by_id("lbl-email");
```
Comment (Harsha509) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.de.md:390]:
```suggestion
emailAddressLabel = driver.find_element(By.ID, "lbl-email");    
```

**Comment by nainappa** [comment]
@Harsha509 Thanks for the review. I am sorry about the Project file deletion. I will make sure, it won't happen again. Regarding second question, I verified them through latest Alpha-5 version.

Regarding the review comment, I initially thought of using find_element. But looks like the usage of  find_element and find_element_by_id depends on the context (Like in Page object pattern we use find_element for the obvious reasons). Also the previous examples were using find_element_by_css. So, thats why I chose to use find_element_by_id. Can you please shed some light on that? I will make the changes and raise a new PR.

**Comment by Harsha509** [comment]
Hi @nainappa ,

Thanks for the conformation on code samples.

Usage of `find_element_by_id`, `find_element_by_css` are going to be deprecated in python bindings. We would like to target the code samples with latest changes. 

Regards,
Harsha.

**Comment by nainappa** [comment]
@Harsha509 . Thanks for letting me know that. I will make the changes and raise PR in sometime.

**Comment by nainappa** [comment]
@Harsha509 I changed all the '_find_element_by_id_' to '_find_element(By.ID,_ '. Please take a look when you have time.

**Pull request merged**